components/dashboard_modules/ExpectedOutput.tsx

```diff
@@ -0,0 +1,43 @@
+import NumberDisplay from './NumberDisplay';
+import { formatter, useFutureGraphData } from 'lib/utils';
+
+interface ForecastDataPoint {
+  target_datetime_utc: number;
+  expected_generation_kw: number;
+}
+
+const computeMipointRiemmanSum = (points: ForecastDataPoint[] | undefined) => {
+  if (!points) {
+    return 'Error';
+  }
+
+  var approxArea = 0.0;
```

Comment:
Prefer `let` and simple `0`.
```suggestion
  let approxArea = 0;
```